.travis.yml

```diff
@@ -37,6 +38,16 @@ script:
 
 # Report coverage
 after_success:
+  - if [[ "${PYVER}" == "3.6" ]];
+    then
+        source activate root;
+        conda remove -y --force -n testenv --all;
+        conda create -y -n testenv python="3.5";
+        conda remove -y --force -n testenv openssl;
+        conda env update -n testenv --file environment_ci.yml;
+        conda list --full-name -n testenv openssl || conda install -y -n testenv openssl;
+        source activate testenv;
+    fi
```

Comment:
This is kind of a hack, but we seemed to have issues getting `coveralls` to upload the results from `coverage`. The bug seems to be in `coverage`, but since it is such an old version I think we will just need to upgrade.